### ISSUE TITLE
Test the stability of ensembles as a function of N_ensembles and hyperparameters 

### ISSUE THREAD

**Issue by maho3** [maho3/ltu-ili#33]
We've had an issue that multiple inference trainings using the same data at the same hyperparameters results in vastly different posterior constraints. We've recently added [network ensembling](https://github.com/maho3/ltu-ili/blob/0943be9deba798c9f8d47497e70f108d0f92950f/ili/inference/runner_sbi.py#L178) to combat this, under the assumption that more neural posteriors will average out any overfitting.

The task here is to run our existing framework on a single dataset at many different N_ensembles and model hyperparameters (architectures, learning rate, etc.) to see if you can get it to consistently converge on a single predictive posterior. It would be nice if you can produce plots of many posteriors and their validation metrics as a function of N_ensembles or e.g. model depth. Also, it would be useful to benchmark how many ensembles do we need in order to get reliable results.

**Comment by maho3** [comment]
If we are really sensitive to hyperparameters, we may need to implement hyperparameter searching soon...
